app/src/test/java/org/simple/clinic/login/applock/BCryptPasswordHasherTest.kt

```diff
@@ -1,6 +1,8 @@
 package org.simple.clinic.login.applock
 
 import org.junit.Test
+import org.simple.clinic.security.BCryptPasswordHasher
+import org.simple.clinic.security.ComparisonResult
 
 class BCryptPasswordHasherTest {
```

Comment:
Shouldn't this test also get moved to the `security` package?

Reply:
Whoops, missed this. Fixing this.